scrapers/ArchAngelVideo.yml

```diff
@@ -7,34 +7,29 @@ sceneByURL:
 xPathScrapers:
   sceneScraper:
     common:
-      $videoContent: //div[@class="content"][1]
+      $videoContent: //div[@class="midwrap clear"]
     scene:
-      Title: $videoContent//h4/a/text()
-      Details: //div[@class="item-info"]/p[@class="description"]
+      Title: $videoContent//div[@class='title clear']/h2
+      Details: //div[@class='description']/p
       Date:
-        selector: $videoContent//ul[@class="item-meta"]//li[1]/text()
-        concat: " "
+        selector: $videoContent//div[@class='info']/p/text()[1]
```

Comment:
maybe just to be safe use 
```yaml
        selector: $videoContent//div[@class='info']/p/text()[contains(.,"Added:")]
```
instead of the first text ?

Reply:
Yeah, that's a better selector, just changed it.